README.md

```diff
@@ -91,3 +91,30 @@ This is suitable for smaller changes or things you can check via the HTML dashbo
 ### Let CI do it
 
 The simpler but less flexible way is to let the [CI](https://gitlab.com/satoshilabs/trezor/trezor-user-env/pipelines) build it. You can create a branch, commit your changes and then push them. The CI will build it for you and tag the appropriate docker image as `test`. You can then modify all scripts/commands and use `registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env:test` instead of `registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env` which defaults to the `latest` tag which equals trezor-user-env's master. Suite's docker-compose files in the `docker` subdirectory are the place where you want to change this.
+
+## Local development against Suite end-to-end tests
+_NOTE: The steps below are all dependant on Docker containers. It could be even better to be able to avoid the Docker completely - just running trezor-user-env on localhost and connecting Suite tests to it - but hard to say how to do it, all my experiments failed. Please feel free to improve the solution._
+
+It may be helpful to run Suite e2e tests locally against the local trezor-user-env state, to avoid the need to involve the CI (Gitlab jobs) and the connected pain of merging changes into one branch just to test them.
+
+#### Suite preparation steps:
+- taken from [this](https://github.com/trezor/trezor-suite/blob/develop/docs/tests/e2e-web.md) and [this](https://github.com/trezor/trezor-suite/#development) Suite guides
+- `git clone git@github.com:trezor/trezor-suite.git`
+- `cd trezor-suite`
+- `sudo yarn`
+- `sudo yarn build:libs && yarn workspace @trezor/suite-data msg-system-sign-config`
+- `xhost +`
+- `docker/docker-suite-install.sh`
```

Comment:
Maybe let's not copy and paste these commands and just link to the Suite's e2e-web.md file? In case it changes in future..

Reply:
These commands are working currently, so I would leave them here, to have everything in one place. We could add a note "in case of changes, the above-mentioned links show the up-to-date steps"

Reply:
I really think it is better to just link the instructions. Because then for example we'll remove `msg-system-sign-config` in Suite and people will be super confused. Let's remove it.